.travis.yml

```diff
@@ -2,13 +2,13 @@ language:
   php
 
 php:
-  - 5.3
-  - 5.4
   - 5.5
+  - 5.6
+  - 7.0
 
 before_script:
   - curl -s http://getcomposer.org/installer | php
-  - php composer.phar update -v
+  - php composer.phar install -v
```

Comment:
install will install the dependencies that are in the lock file. update will check for new versions of the dependencies and update the lock file. In travis you want to test what's in the lock file, because thats ultimately what will be installed.